cosmwasm/submit-proposal.js

```diff
@@ -49,7 +52,32 @@ const storeCode = (client, wallet, config, options) => {
             printInfo('Proposal submitted', proposalId);
 
             contractConfig.storeCodeProposalId = proposalId;
+            contractConfig.storeCodeProposalCodeHash = createHash('sha256').update(readWasmFile(options)).digest().toString('hex');
+        }
+    });
+};
+
+const fetchCodeId = (client, contractConfig, { fetchCodeId }) => {
+    if (!fetchCodeId) {
+        return Promise.resolve();
+    }
+
+    return client.getCodes().then((codes) => {
```

Comment:
ah yes, forgot to add a TODO in there, we should create our own implementation to make it more efficient. For now I chose to leave it like that since at the beginning we are probably not going to have many codeIds